encoding.bs

```diff
@@ -2650,6 +2650,15 @@ consumers of content generated with <a>GBK</a>'s <a for=/>encoder</a>.
 
 <h4 id=iso-2022-jp-encoder dfn export>ISO-2022-JP encoder</h4>
 
+<div class="note no-backref">
+ <p>The <a>ISO-2022-JP encoder</a> is the only <a for=/>encoder</a> for which the concatenation of
+ multiple outputs can result in an <a>error</a> when run through the <a>ISO-2022-JP decoder</a>.
```

Comment:
I'd say "corresponding decoder" instead of "ISO-2022-JP decoder" to be pedantic about this being self-decode and not about decoding the output of other encoders with the ISO-2022-JP decoder. Other than that, looks good to me.